README.md

```diff
@@ -40,48 +86,43 @@ this behaviour, add the following line to the very top of your `Makefile`:
 DISABLE_GITHOOKS=true
 ```
 
-## Make tasks
-
-See in [index.mk](index.mk) for all the different tasks you can use in your `Makefile`.
-
-## CLI
-
-This includes a CLI for you to use to do some things.
-
-### get-config
+### CLI
 
 This tool helps you to obtain configuration for your project.
 
 ```sh
-$ ngage
-ngage get-config --help
+$ ngage get-config
+
+get environment variables from Vault
```

Comment:
Does this require you to be logged in to Vault? If so then might be worthwhile including a link to the Vault docs to benefit new starters.

Reply:
it does 👍

Reply:
I can't see one in the README. Might be worth including there to make obvious.